### PR TITLE
Add missing constant in config for local development

### PR DESCRIPTION
**Pull request by parostatkiem**
**Description**

Changes proposed in this pull request:

- Fix error that prevented AddOns from rendering when run not on cluster
- ...
- ...

**Related issue(s)**

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -25,4 +25,12 @@ window.clusterConfig = {
   addOnsModuleUrl: 'http://' + localDomain + ':8004',
   lambdasModuleUrl: 'http://' + localDomain + ':4201',
   logsModuleUrl: 'http://' + localDomain + ':4400',
+
+  HELM_BROKER_REPO_URL_PREFIXES: [
```
Comment (dariadomagala) [inline config.js]: all variables are camel case, could we also do it here? ;)

Comment (parostatkiem) [reply config.js]: No. We would have to change it deep inside Kyma :(

Comment (magicmatatjahu) [inline config.js]: Please add this table [here](https://github.com/kyma-project/console/blob/master/add-ons/src/constants.ts#L5)

Comment (parostatkiem) [reply config.js]: Why?

**Pull request merged**